src/_help/enhance-documentation-content/markdown-support.md

```diff
@@ -106,8 +110,8 @@ At least one `height` *or* `width` parameter is mandatory, everything else being
 =x50px        # without width
 ```
 
->- If you don't specify a `unit` it will default to pixel
-- If you don't specify `width` *or* `height`, the other value will be a ratio calculated from the original size of the image so it doesn't shrink
+>- If you don't specify a `unit` it will default to pixel.
+- If you don't specify `width` *or* `height`, the other value will be a ratio calculated from the original size of the image so it doesn't shrink.
 {: .info}
```

Comment:
How about writing an example for each supported case, so that everything is crystal clear? Maybe it's just me, but I'm now a little confused about how exactly I should write the "dimension" part. I'd recommend that the example shows the whole parenthesis, including the path, "image title", and so on.

Reply:
Don't we have that example on line 94? Or would you like something else?